go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/Azure/azure-storage-azcopy
 require (
 	github.com/Azure/azure-pipeline-go v0.2.1
 	github.com/Azure/azure-storage-blob-go v0.7.0
-	github.com/Azure/azure-storage-file-go v0.6.1-0.20191219012846-66d72a9c7823
+	github.com/Azure/azure-storage-file-go v0.6.1-0.20200306202433-9355d1265351
```

Comment:
Will this be the final version number? Or is there an update to follow once the storage-file-go PR is merged?

Reply:
There will need to be an update once storage-file-go is merged.